static/app/views/relocation/relocation.spec.tsx

```diff
@@ -1,10 +1,46 @@
 import {initializeOrg} from 'sentry-test/initializeOrg';
-import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+import {render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrary';
 
 import ConfigStore from 'sentry/stores/configStore';
 import Relocation from 'sentry/views/relocation/relocation';
 
+const fakePublicKey = `-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAw5Or1zsGE1XJTL4q+1c4
+Ztu8+7SC/exrnEYlWH+LVLI8TVyuGwDTAXrgKHGwaMM5ZnjijP5i8+ph8lfLrybT
+l+2D81qPIqagEtNMDaHqUDm5Tq7I2qvxkJ5YuDLawRUPccKMwWlIDR2Gvfe3efce
+870EicPsExz4uPOkNXGHJZ/FwCQrLo87MXFeqrqj+0Cf+qwCQSCW9qFWe5cj+zqt
+eeJa0qflcHHQzxK4/EKKpl/hkt4zi0aE/PuJgvJz2KB+X3+LzekTy90LzW3VhR4y
+IAxCAaGQJVsg9dhKOORjAf4XK9aXHvy/jUSyT43opj6AgNqXlKEQjb1NBA8qbJJS
+8wIDAQAB
+-----END PUBLIC KEY-----`;
+
 describe('Relocation', function () {
+  let fetchPublicKey: jest.Mock;
+
+  beforeEach(function () {
+    MockApiClient.asyncDelay = undefined;
+    MockApiClient.clearMockResponses();
+    fetchPublicKey = MockApiClient.addMockResponse({
+      url: '/publickeys/relocations/',
+      body: {
+        public_key: fakePublicKey,
+      },
+    });
+
+    // The tests fail because we have a "component update was not wrapped in act" error. It should
+    // be safe to ignore this error, but we should remove the mock once we move to react testing
+    // library.
+    //
+    // eslint-disable-next-line no-console
+    jest.spyOn(console, 'error').mockImplementation(jest.fn());
+  });
+
+  afterEach(function () {
```

Comment:
I think it's good practice to reset state before and after. For your tests you should only need to do it before, but it's likely that someone else relies on some state being "clean" in some other test somewhere, and have not cleaned it in *their* before handler. Since test order can be random, clearing before and after is a defense against off flakes due to that assumption.